src/amr/data/particles/refine/split.h

```diff
@@ -24,8 +78,42 @@ namespace amr
 
 #define ISNOTTABULATED(dim, RF) (dim != 1 || RF != 2)
 
+
+    template<std::size_t dimension, std::size_t interpOrder>
+    class _Split;
+
+    class SuperSplit
+    {
+    public:
+        template<std::size_t dimension, std::size_t interpOrder>
+        using Super = typename std::conditional<
```

Comment:
oops, syntax far too complicated for me to understand...

Reply:
this picks a default implementation per dimension and isn't going to survive so not that important